functions/Remove-DbaDbView.ps1

```diff
@@ -0,0 +1,118 @@
+function Remove-DbaDbView {
+    <#
+    .SYNOPSIS
+        Removes a database view from database(s) for each instance(s) of SQL Server.
+
+    .DESCRIPTION
+        The Remove-DbaDbView removes view(s) from database(s) for each instance(s) of SQL Server.
+
+    .PARAMETER SqlInstance
+        The target SQL Server instance or instances. This can be a collection and receive pipeline input to allow the function to be executed against multiple SQL Server instances.
+
+    .PARAMETER SqlCredential
+        Login to the target instance using alternative credentials. Accepts PowerShell credentials (Get-Credential).
+
+        Windows Authentication, SQL Server Authentication, Active Directory - Password, and Active Directory - Integrated are all supported.
+
+        For MFA support, please use Connect-DbaInstance.
+
+    .PARAMETER Database
+        The database(s) to process. This list is auto-populated from the server. If unspecified, all databases will be processed.
+
+    .PARAMETER ExcludeDatabase
+        The database(s) to exclude. This list is auto-populated from the server.
+
+    .PARAMETER View
+        The view(s) to process. If unspecified, all views will be processed.
+
+    .PARAMETER IncludeSystemDbs
```

Comment:
You don't use this parameter at the moment.

Reply:
Yeah, the same about "Schema" was there originally, but the Get-DbaDbView does not have it either. Wanted to finish the Remove- first so it is satisfactory and then add more options to the Get-DbaDbView as well.

Reply:
There is no reason to use this as the assumption is the user has decided either from providing the database name or piping what databases will be included or excluded.